### PR TITLE
Rename channel particicpation CFT filerepo directory to pendingops

### PR DESCRIPTION
**Pull request by jcastrence**
#### Type of change

- Improvement (improvement to code, performance, etc)

#### Description

- Renamed the filerepo directory associated with storing pending operations for channel participation CFT to pendingops for better context

#### Related issues

[FAB-18280](https://jira.hyperledger.org/browse/FAB-18280)

### CODE REVIEW

File: common/ledger/blockledger/fileledger/factory_test.go [hunk 1]
```diff
@@ -222,7 +222,7 @@ func TestRemove(t *testing.T) {
 		err = f.Remove("foo")
 		require.EqualError(t, err, "oogie")
 
-		dest := filepath.Join(dir, "filerepo", "remove", "foo.remove")
+		dest := filepath.Join(dir, "pendingops", "remove", "foo.remove")
 		_, err = os.Stat(dest)
 		require.NoError(t, err, "Expected foo.remove to exist")
 	})
```
Comment (wlahti) [inline common/ledger/blockledger/fileledger/factory_test.go:228]: Missed the rename in the test case right below this one.

**Review by wlahti — COMMENTED** [review]
Almost got all of them on your first pass. :) Looks like you missed the one I noted in one of the files and then also [here](https://github.com/hyperledger/fabric/blob/master/orderer/common/server/etcdraft_test.go#L163).

File: common/ledger/blockledger/fileledger/factory_test.go (hunk 1 above)
Comment (jcastrence) [reply common/ledger/blockledger/fileledger/factory_test.go:228]: Thanks for catching that

Comment (jcastrence) [reply common/ledger/blockledger/fileledger/factory_test.go:228]: Fixed

**Review by lindluni — APPROVED** [review]

File: common/ledger/blockledger/fileledger/factory_test.go [hunk 2]
```diff
@@ -222,15 +222,15 @@ func TestRemove(t *testing.T) {
...
 
-	t.Run("saving to file repo fails", func(t *testing.T) {
+	t.Run("saving to pending ops fails", func(t *testing.T) {
```
Comment (wlahti) [inline common/ledger/blockledger/fileledger/factory_test.go:230]: Super nit: this didn't need to change and confuses things slightly but it doesn't matter. :) We're still using the filerepo as the underlying implementation (and that's what's failing to save for this test), it's just the public naming of the folder that changed.

**Review by wlahti — APPROVED** [review]
One extremely minor nit in the naming of a test case but otherwise looks good. Thanks, Julian!

**Pull request merged**